doc/src/base/scopedvalues.md

```diff
@@ -0,0 +1,138 @@
+# Scoped Values
+
+Scoped values provide an implementation of dynamic scoping in Julia.
+Dynamic scope means that the state of the value is dependent on the execution path
+of the program. This means that for a scoped value you may observe
+multiple different values at the same time.
+
+!!! compat "Julia 1.11"
+    Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible
```

Comment:
The docstring fro ScopedValue mentions a compatible version for Julia _1.7_, so should that docstring change? Or this one?

Reply:
It's `1.8` due to the usage for `const` on fields.